.gitlab-scripts/setup_artifactory.cmd

```diff
@@ -1,4 +1,4 @@
 if exist %APPDATA%\pip rd /s/q %APPDATA%\pip
 mkdir %APPDATA%\pip
 echo [global] > %APPDATA%\pip\pip.ini
-echo extra-index-url = https://%ARTIFACTORY_USER%:%ARTIFACTORY_PASSWORD%@%ARTIFACTORY_URL%/simple >> %APPDATA%\pip\pip.ini
+echo extra-index-url = https://%artifactory_user%:%artifactory_password%@%ARTIFACTORY_PYPI_URL% >> %APPDATA%\pip\pip.ini
```

Comment:
Does the pip.ini file live on public infrastructure? If so, does it get destroyed when it is not needed anymore?

Reply:
this runs it in the gitlab runner, check the gitlab-ci-agent